src/developer-testing/developer-testing.service.ts

```diff
@@ -0,0 +1,172 @@
+/* istanbul ignore file */
```

Comment:
Why are we ignoring the testing for this file? Atleast we should create a task to put in the tests later if we want this urgently. Not sure if I'm missing something here